.env.production

```diff
@@ -8,3 +8,5 @@ DFE_SIGNIN_API_SCHOOL_CHANGE_ENABLED="1"
 GTM_ID=GTM-W3VGWP6
 GTM_UA_ID=UA-136255210-1
 BIGQUERY_DATASET=gse_events_production
+
+DEGREE_SUBJECT_AUTOCOMPLETE=0
```

Comment:
Just for consistency with the flags above, should this be called `DEGREE_SUBJECT_AUTOCOMPLETE_ENABLED` with a value of "0" or "1"

Reply:
yes, good point - I'll rename it - thanks 👍

Reply:
@asatwal now renamed to `DEGREE_SUBJECT_AUTOCOMPLETE_ENABLED`